class UserError(Exception):
    pass


class APIError(Exception):
    pass

